tests/storage/hotplug.go

```diff
@@ -829,48 +829,6 @@ var _ = SIGDescribe("Hotplug", func() {
 				vmi = tests.RunVMIAndExpectLaunch(vmi, 240)
 			})
 
-			It("should mark VMI failed, if an attachment pod is deleted", func() {
```

Comment:
we need to add a functional test that validates that the old attachment pods exit as expected as new pods are created during hot plug

Reply:
Sure I can add such a test.